src/Swashbuckle.AspNetCore.SwaggerGen/SchemaGenerator/SchemaGenerator.cs

```diff
@@ -392,7 +393,13 @@ private OpenApiSchema CreateObjectSchema(DataContract dataContract, SchemaReposi
                     ? GenerateSchemaForMember(dataProperty.MemberType, schemaRepository, dataProperty.MemberInfo, dataProperty)
                     : GenerateSchemaForType(dataProperty.MemberType, schemaRepository);
 
-                if ((dataProperty.IsRequired || customAttributes.OfType<RequiredAttribute>().Any())
+                if ((
+                    dataProperty.IsRequired
+                    || customAttributes.OfType<RequiredAttribute>().Any()
+#if NET7_0_OR_GREATER
+                    || customAttributes.OfType<RequiredMemberAttribute>().Any()
```

Comment:
As this is only used in this single place, removing the `using` and fully-qualifying the type means we don't have a seemingly unused `using` for other targets.

```suggestion
                    || customAttributes.OfType<System.Runtime.CompilerServices.RequiredMemberAttribute>().Any()
```